README.md

```diff
@@ -289,6 +292,14 @@ expect(getByTestId('count-value')).not.toHaveTextContent('21')
 // ...
 ```
 
+## Other
+
+#### `waitForExpect(expectation: () => void, timeout?: number, interval?: number) => Promise<{}>;`
+
+When in need to wait for non-deterministic periods of time you can use waitForExpect,
+to wait for your expectations to pass. Take a look at [`Is there a different way to wait for things to happen?`](#waitForExcept) part of the FAQ,
+or the function documentation here: [`wait-for-expect`](https://github.com/TheBrainFamily/wait-for-expect)
```

Comment:
Could we add a very simple (incomplete) example of how to use the function? We can keep the FAQ too, but a simple example will help people know what this does:

```js
// ...
await waitForExpect(() => expect(queryByLabelText('username')).not.toBeNull())
getByLabelText('username').value = 'chucknorris'
// ...
```